.eslintrc.json

```diff
@@ -20,6 +20,7 @@
     "node_modules/"
   ],
   "rules": {
+    "@typescript-eslint/no-explicit-any": ["off"],
```

Comment:
Is the rationale that if an _explicit_ `any` is required, then that's allowed?  That the developer would need to consciously and judiciously add the `any` declaration, knowing there was no way around it?  I had to do it once in the case of `BlissSymbol.ts`.  In that case I disabled the check for explicit `any` but just for that one line.  Are there are a lot of places where explicit `any` is used?

Also, why is it not just "off"?  Why the array?

Reply:
Good catch, @klown. Allowing the use of "any" type across the project is dangerous. It's better to do it case by case as what you did with `BlissSymbol.ts`. I will remove it. I used "any" when working on this PR but removed them at the end. This line is no longer needed.